src/shared/model/OncoKB.ts

```diff
@@ -28,3 +28,8 @@ export interface IOncoKbData {
     indicatorMap: {[id:string]: IndicatorQueryResp} | null;
     sampleToTumorMap: {[sampleId:string]: string} | null;
 }
+
+export interface IOncoKbDataWrapper {
```

Comment:
Actually this is a simplified version of `MobXPromise` type. The only reason that I added this type is that I don't want lower level components to have dependency on the actual `MobXPromise` type.

This may not be the best name for this type though. Any other suggestions are welcome.